build.gradle

```diff
@@ -68,9 +68,7 @@ dependencies {
     compile 'org.mindrot:jbcrypt:0.3m'
     compile 'org.yaml:snakeyaml:1.17'
     compile 'org.json:json:20160810'
-
-    compile files('lib/AppleJavaExtensions-1.6.jar')
```

Comment:
One question - AppleJavaExtensions was being used for UPNP? Or is it just unused?